SConstruct

```diff
@@ -178,8 +178,21 @@ opts.Add(
         "optimize", "Optimization level", "speed_trace", ("none", "custom", "debug", "speed", "speed_trace", "size")
     )
 )
-opts.Add(BoolVariable("debug_symbols", "Build with debugging symbols", False))
-opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a separate file", False))
+opts.Add(
+    EnumVariable(
+        "debug_symbols",
+        "Build with debugging symbols (options are no, embedded, and separate, but yes is still supported for backwards compatibility)",
```

Comment:
The options don't need to be mentioned here, as validation will mention the valid enum values automatically.